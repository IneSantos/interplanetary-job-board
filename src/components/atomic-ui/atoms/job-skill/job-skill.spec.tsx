import React from "react";
import { render } from '@testing-library/react';
import JobSkill from "./job-skill";

describe("JobSkill atom tests", () => {
  test(`JobSkill component render`, () => {
    const { container } = render(
      <JobSkill skill="This is a skill"/>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
