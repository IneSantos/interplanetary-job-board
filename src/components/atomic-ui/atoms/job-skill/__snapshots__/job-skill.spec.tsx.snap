// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JobSkill atom tests JobSkill component render 1`] = `
<span
  class="skill"
>
  This is a skill
</span>
`;
